app/help/en/index.md

```diff
@@ -1,9 +1,15 @@
-# Graphene Help
+# Help
 
-BitShares 2.0 is a Financial Smart Contracts platform that is based on the
-[Graphene](https://github.com/cryptonomex/graphene) technology built by
-[Cryptonomex](http://cryptonomex.com). You can see Graphene as a toolkit or
-technology for real-time blockchains.
+This is the reference wallet of BitShares Blockchain which connects to BitShares Blockchain nodes (bitshares-core).
+
+This reference wallet is a browser based wallet, which means your keys are stored locally in your browser and never leaves it while using the wallet. You should never expose your keys to anyone as transactions are signed locally before transmitting them to a node.
+
+BitShares Blockchain Reference wallet users can:
+
+- Login to BitShares Blockchain with their private keys to send and receive core cryptocurrency BTS.
```

Comment:
You can't really login to a blockchain, we should brainstorm what wording is appropriate

Reply:
Perhaps replace with "login to BitShares Blockchain node (bitshares-core)"? please brainstorm or propose where/how to brainstorm.